adapters/rest-adapter-generator/src/test/java/uk/gov/justice/api/resource/DefaultUsersUserIdResourceIT.java

```diff
@@ -200,7 +203,7 @@ public void shouldReturnResponseWithContentType() {
                 .path("/users/4444-5556")
                 .header("Accept", "application/vnd.people.queries.get-user+json")
                 .get();
-
+        assertThat(response.getStatus(), is(200));
```

Comment:
Bad 200 magic number, use Http Response Success Constant (and replace elsewhere also)